tests/clims/models/test_substance.py

```diff
@@ -107,6 +107,64 @@ def do_asserts(substance):
         fresh_substance = Substance.objects.get(name=substance.name)
         do_asserts(self.app.substances.to_wrapper(fresh_substance))
 
+    @pytest.mark.now
+    def test_create_pool__from_two_aliquots__pool_origin_are_original_samples(self):
+        # Arrange
+        props = dict(preciousness='*o*', color='red')
+        sample1 = self.create_gemstone(**props)
+        sample2 = self.create_gemstone(**props)
+        aliquot1 = sample1.create_child()
+        aliquot2 = sample2.create_child()
+
+        # Act
+        pool = GemstoneSample(
```

Comment:
Perhaps we want to look at the terminology here? Aliquots and pools may not make that much sense for gemstones? Perhaps we want to call it something like `slice` and `combo` instead? While it seems like a minor thing, I think that keeping the terminology clear of liquid based lab stuff will help us avoid making assumptions that will make life difficult for us, later down the road.

Reply:
ok.